frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -70,7 +70,7 @@ function mapStateToProps(state: State, { fields = [] }: { fields: Field[] }) {
 }
 
 export interface IFieldValuesWidgetProps {
-  color?: string;
+  color?: "brand";
```

Comment:
Yeah, this one drove me nuts yesterday. So there's only two options it could possibly be - `brand` and `purple`. _but_ I was going through the classes and it seemed like we didn't ever have a `text-purple` or `text-purple-hover` class, which just meant that `purple` meant `no class`. So that's why it's just `brand`